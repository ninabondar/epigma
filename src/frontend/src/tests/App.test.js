import React from "react"
import App from "../components/App"

describe("<App/>", () => {
  test("should match its snapshot", () => {
    const component = shallow(<App />)
    expect(component).toMatchSnapshot()
  })
})
