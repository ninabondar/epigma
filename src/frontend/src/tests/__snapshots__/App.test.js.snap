// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> should match its snapshot 1`] = `
<Fragment>
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(withProps(withState(withHandlers(branch(branch(DocumentsListFetched))))))",
        "type": [Function],
      }
    }
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/login"
  />
  <Route
    component={[Function]}
    path="/edit/:documentId"
  />
</Fragment>
`;
